src/components/ProposalApplication.tsx

```diff
@@ -8,13 +13,19 @@ interface ProposalApplicationProps {
   proposalDetails: ProposalDetails
   isStudent: boolean
   isSupervisor: boolean
+  refetch: () => void
 }
 
 export default function ProposalApplication({
   proposalDetails,
   isStudent,
   isSupervisor,
+  refetch,
 }: ProposalApplicationProps) {
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(true)
+
+  const acceptApplication = trpc.acceptProposalApplication.useMutation()
+
   if (proposalDetails?.typeKey === 'SUPERVISOR') {
     return (
       <div className="p-4">
```

Comment:
one can currently see applications on all proposals. we need to restrict it such that only applications to owned proposals are shown, so that no applications are leaked to other supervisors, and so that supervisors can only accept applications on their own proposals. -> show the entire applications area only on owned proposals